Classes/Fluid/Exception/SchemaException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace SMS\FluidComponents\Fluid\Exception;
+
+class SchemaException extends \TYPO3Fluid\Fluid\Core\ViewHelper\Exception
```

Comment:
I prefer a import and the use of the use statement over the FQN. Here and in the following.